ui/app/templates/vault/cluster/init.hbs

```diff
@@ -157,4 +149,4 @@
       </form>
     </Page.content>
   {{/if}}
-</SplashPage>
+</SplashPage>
```

Comment:
Intentional lack of newline or fallout from switching to VSCode 🤔

Reply:
😱 also another VSCode thing which I think I've fixed...